.pre-commit-config.yaml

```diff
@@ -27,6 +27,6 @@ repos:
     name: mypy
     stages: [commit]
     language: system
-    entry: poetry run mypy **/*.py
+    entry: poetry run mypy pykeybasebot/
```

Comment:
I was getting "No directory found" for **/*.py. Also I think this should match with what is in the Makefile for `make test`

Reply:
we could do `poetry run mypy pykeybasebot examples tests` (though i think i was getting errors with the `tests` dir)